src/extension/panel.ts

```diff
@@ -124,7 +124,13 @@ export class Panel {
                 }
                 case 'select': {
                     const {logUri, uri, region} = message as { logUri: string, uri: string, region: Region};
```

Comment:
friendly ping @EasyRhinoMSFT

Reply:
> CC @EasyRhinoMSFT, this might not be a block for this PR, and would therefore need an Issue to track the fix.

OK I'll approve, please open an issue.